core/src/main/scala/org/apache/spark/h2o/utils/ReflectionUtils.scala

```diff
@@ -168,6 +170,14 @@ object ReflectionUtils {
     }
   }
 
+  def supportedType(columnType: String): SupportedType = columnType match {
```

Comment:
I would maybe stress that this method converts H2O type to our supported type, what do you think?